<launch>
  
  <arg name="rawlog_file" default="$(find mrpt_rawlog)/tutorial//RO-SLAM_demo.rawlog" />
  <arg name="rate" default="10" />

  <node pkg="mrpt_rawlog" type="rawlog_play_node" name="rawlog_play_node" output="screen">

    <param name="debug" value="true"/> 
    <param name="rawlog_file" value="$(arg rawlog_file)"/>
    <param name="rate" value="$(arg rate)"/>

  </node>

</launch>

